standalone/soroban-rpc/etc/soroban-rpc.cfg

```diff
@@ -0,0 +1,13 @@
+ENDPOINT="localhost:8003"
+HORIZON_URL="http://localhost:8001"
+FRIENDBOT_URL="http://localhost:8000/friendbot"
+NETWORK_PASSPHRASE="__NETWORK__"
+STELLAR_CORE_URL="http://localhost:11626"
+CAPTIVE_CORE_CONFIG_PATH="/opt/stellar/soroban-rpc/etc/stellar-captive-core.cfg"
+CAPTIVE_CORE_STORAGE_PATH="/opt/stellar/soroban-rpc/captive-core"
+CAPTIVE_CORE_USE_DB="true"
+STELLAR_CORE_BINARY_PATH="/usr/bin/stellar-core"
+HISTORY_ARCHIVE_URLS=["__ARCHIVE__"]
```

Comment:
That would be nice. But in `start` the HISTORY_ARCHIVE_URLS is also used to swap in the horizon.env, which expects comma-separated. We also only use a single URL for now, so...

Reply:
ok, yes, since only at single value'd archive urls, can be as-is, just for consideration, could put a `TODO` in here for reminder to refactor `start` and .cfg  and .env template files to have consistent usage around the urls substitution, I think it makes sense for `start` to encode array brackets as needed rather than the templates, but that is just opinion or part of the `TODO`, thanks!